docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -168,70 +168,64 @@ For `elasticsearch` the following arguments are supported:
-In order to deploy the Elastic Serverless Forwarder for {aws} you must define some parameters, that will be used by every of the methods available for deployment.
-
-There are four different groups of parameters.
-
-- General Configuration: parameters related to the general configuration and the Elastic Serverless Forwarder behaviour
-- Inputs: parameters related to your specific <<aws-serverless-forwarder-inputs>> (ie: event triggers)
-- S3 Buckets Permissions: parameters related to the permissions required in order to access the S3 Buckets of <<aws-serverless-forwarder-inputs-s3>>
-- Network: permissions related to the network environment of the Elastic Serverless Forwarder
-
-- General Configuration:
-  * `ElasticServerlessForwarderS3ConfigFile`
-  * `ElasticServerlessForwarderSSMSecrets`
-  * `ElasticServerlessForwarderKMSKeys`
-- Event Triggers:
-  * `ElasticServerlessForwarderSQSEvents`
-  * `ElasticServerlessForwarderS3SQSEvents`
-  * `ElasticServerlessForwarderKinesisEvents`
-  * `ElasticServerlessForwarderCloudWatchLogsEvents`
-- S3 Buckets Permissions:
-  * `ElasticServerlessForwarderS3Buckets`
-- Network:
-  * `ElasticServerlessForwarderSecurityGroups`
-  * `ElasticServerlessForwarderSubnets`
-
-Parameters:
-
-- `ElasticServerlessForwarderS3ConfigFile`: Set this value to the location of your Elastic Serverless Forwarder configuration file in S3 URL format: `s3://bucket-name/config-file-name`. This will populate the `S3_CONFIG_FILE` environment variable for the forwarder.
+== Define deployment parameters
+Whichever deployment method you choose, you must define the following parameters correctly for your setup. This section explains the types of parameters and provides guidance on how to set them to match your deployment(s).
+
+=== General configuration
+These parameters define the general configuration and behaviour for the forwarder.
+
+- `ElasticServerlessForwarderS3ConfigFile`: Set this value to the location of your `config.yaml` in S3 URL format: `s3://bucket-name/config-file-name`. This will populate the `S3_CONFIG_FILE` environment variable for the forwarder.
 - `ElasticServerlessForwarderSSMSecrets`: Add a comma delimited list of {aws} SSM Secrets ARNs used in the `config.yml` (if any).
-- `ElasticServerlessForwarderKMSKeys`: Add a comma delimited list of {aws} KMS Keys ARNs to be used for decrypting {aws} SSM Secrets, Kinesis Data Streams or SQS queue (if any).
-- `ElasticServerlessForwarderSQSEvents`: Add a comma delimited list of Direct SQS queues ARNs to set as event triggers for the forwarder (if any).
+- `ElasticServerlessForwarderKMSKeys`: Add a comma delimited list of {aws} KMS Keys ARNs to be used for decrypting {aws} SSM Secrets, Kinesis Data Streams, or SQS queue (if any).
+- `ElasticServerlessForwarderAssumeRoles`: Add a comma delimited list of {aws} IAM Roles ARNs that the forwarder should be given permission for the `sts:AssumeRole` action (if any).
```

Comment:
this must be removed: it is not needed.
probably a leftover since you started the review and I pushed the latest change